client/.env.dist

```diff
@@ -1,14 +0,0 @@
-# Api settings
```

Comment:
Why do you remove this file (and the one from the server) ? I think it still a good idea to have a template when in development or may be if someone wants a custom install without docker for instance, what do you think?

Reply:
hmm, great point!